com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/JdtlsUtils.java

```diff
@@ -161,19 +166,6 @@ public static ISourceRange getNameRange(IJavaElement element) throws JavaModelEx
 		return nameRange;
 	}
 
-	/**

```

Comment:
I know it's originally from here, and what's the purpose of moving it to lsifUtil?

Reply:
`JdtlsUtils` is to out the utilities referenced in `JDTUtils`

`lsifUtil` is to put general utilities used by the indexer